clients/include/testing_trtri_strided_batched.hpp

```diff
@@ -0,0 +1,216 @@
+/* ************************************************************************
+ * Copyright 2018-2019 Advanced Micro Devices, Inc.
+ * ************************************************************************ */
+
+#include "cblas_interface.hpp"
+#include "flops.hpp"
+#include "near.hpp"
+#include "norm.hpp"
+#include "rocblas.hpp"
+#include "rocblas_init.hpp"
+#include "rocblas_math.hpp"
+#include "rocblas_random.hpp"
+#include "rocblas_test.hpp"
+#include "rocblas_vector.hpp"
+#include "unit.hpp"
+#include "utility.hpp"
+
+template <typename T>
+void testing_trtri_strided_batched(const Arguments& arg)
+{
+    rocblas_int N           = arg.N;
+    rocblas_int lda         = arg.lda;
+    rocblas_int batch_count = arg.batch_count;
+
+    rocblas_int bsa    = lda * N;
+    size_t      size_A = size_t(bsa) * batch_count;
+
+    char char_uplo = arg.uplo;
+    char char_diag = arg.diag;
+
+    // char_uplo = 'U';
+    rocblas_fill     uplo = char2rocblas_fill(char_uplo);
+    rocblas_diagonal diag = char2rocblas_diagonal(char_diag);
+
+    rocblas_local_handle handle;
+
+    // argument sanity check, quick return if input parameters are invalid before allocating invalid
+    // memory
+    if(N < 0 || lda < 0 || lda < N || batch_count <= 0)
+    {
```

Comment:
Not sure why `batch_count == 0` is handled here.

The main code should work with `batch_count == 0`, maybe with guards to make sure that a null pointer allocated with `batch_count` is only really null if `batch_count != 0`.

Reply:
Got rid of the batch_count == 0 check here, the rest of the code already seems to work with batch_count == 0. However it doesn't really do anything interesting as none of the copies do anything (meaning the output of rocblas_trtri isn't checked) and cblas doesn't run with the current code.

Reply:
Removing from here doesn't really help make any additional checks. Do we want to make it a special case and verify that the destination is not being written to? The same goes for the other BLAS1 PRs, otherwise all are just checking for rocblas_status_success as is.